test/testdata/rewriter/minitest_tables.rb.rewrite-tree.exp

```diff
@@ -512,9 +518,7 @@ class <emptyTree><<C <root>>> < (::<todo sym>)
     end
 
     <self>.test_each([]) do |val|
-      <self>.it() do ||
-        <self>.puts("anonymous it blocks")
-      end
+      <runtime method definition of <it>>
```

Comment:
We could likely make the optimization apply for methods defined in `test_each` blocks, but I want to save that for another change.